flowmachine/flowmachine/features/subscriber/handset_stats.py

```diff
@@ -157,7 +154,8 @@ def _make_query(self):
             """
             return sql
 
-        sql = f"""
+        # Weighted median
+        return f"""
```

Comment:
Might be safer to stick this in a `if self.statistic in {"median"}`. As things stand, if we add a new entry to the `Statistic` enum then `HandsetStats` will accept the new stat without complaint, but will calculate the median.